esm/inverse_folding/gvp_transformer.py

```diff
@@ -118,6 +118,10 @@ def sample(self, coords, partial_seq=None, temperature=1.0, confidence=None, dev
         # Run encoder only once
         encoder_out = self.encoder(batch_coords, padding_mask, confidence)
         
+        #Make sure all tensors are on the same device if a GPU is present
```

Comment:
Nit: Add a space between the `#` and the comment to match style

Reply:
@brianhie  Done! 

Scoring and sampling should work now - my setup wasn't catching the error too which I didn't realize until I investigated the linked issue yesterday :)